src/styles/RecipeFinder.css

```diff
@@ -0,0 +1,82 @@
+.dish-input {
+  padding: 0.2rem;
+  border: 0;
+  font-size: 1.3rem;
+  &:focus {
+    border: 0
+  }
+}
+
+.dish-input:focus-visible {
+  outline: none;
+}
+
+.find-recipe-btn {
+  font-size: 1.2rem;
+  border-radius: 0.75rem;
+  cursor: pointer;
+  background-color: rgb(86, 155, 59);
+  color: white;
+  border: 0.1rem solid rgb(214, 209, 209);
+  padding: 0.4rem;
+  padding-left: 0.7rem;
+  padding-right: 0.7rem;
+
+  font-family: "Inter", sans-serif;
+}
+
+.find-recipe-btn:disabled {
+  color: rgb(230, 228, 228);
+  border: 0.1rem solid rgb(160, 159, 159);
+  background-color: rgb(146, 160, 140);
+  cursor: auto;
+}
+
+.input-with-btn {
+  border: 0.1rem solid rgb(214, 209, 209);
+  padding: 0.4rem;
+  border-radius: 0.75rem;
+  display: flex;
+}
+
+.container {
+  min-height: 80%;
+  padding-top: 6rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.form {
```

Comment:
You use this exact same set of rules on the above `.container` class as well. Maybe just generalize this class name and use it for both things?